src/poker/EquityEvaluator.java

```diff
@@ -40,52 +30,65 @@ public HashMap<String, String> loadHashMapFromFile(String filePath) {
-                    for (int l = k + 1; l < 8   ; l++){
-                        List<Card> player1Hand = new ArrayList<>();
-                        List<Card> player2Hand = new ArrayList<>();
-                        player1Hand.add(deck.cardList.get(i));
-                        player1Hand.add(deck.cardList.get(j));
-                        player2Hand.add(deck.cardList.get(k));
-                        player2Hand.add(deck.cardList.get(l));
-                        player1Hand.sort(Comparator.comparing(Card::toString));
-                        player2Hand.sort(Comparator.comparing(Card::toString));
-
-                        List<Card> bothPlayers = new ArrayList<>(player1Hand);
-                        bothPlayers.addAll(player2Hand);
-                        List<Card> bothPlayers2 = new ArrayList<>(player2Hand);
-                        bothPlayers2.addAll(player1Hand);
-
-                        String equity = calculateEquityTest(player1Hand, player2Hand).toString();
-
-                        instaResult.put(bothPlayers.toString(), equity);
-                        instaResult.put(bothPlayers2.toString(),equity);
-                    }
-                }
-            }
-        }
-        saveHashMapToFile(instaResult, "equity_data.ser");
-        return instaResult;
-    }
-    public void getEquity(List<Card> h1, List<Card>h2){
+//    public HashMap<String, String> instantlyResult() {
+//        HashMap<String, String> instaResult = new HashMap<>();
+//        for (int i = 0; i < 8; i++) {
+//            for (int j = i + 1; j < 8; j++) {
+//                for (int k = j + 1; k < 8; k++) {
+//                    for (int l = k + 1; l < 8   ; l++){
+//                        List<Card> player1Hand = new ArrayList<>();
+//                        List<Card> player2Hand = new ArrayList<>();
+//                        player1Hand.add(deck.cardList.get(i));
+//                        player1Hand.add(deck.cardList.get(j));
+//                        player2Hand.add(deck.cardList.get(k));
+//                        player2Hand.add(deck.cardList.get(l));
+//                        player1Hand.sort(Comparator.comparing(Card::toString));
+//                        player2Hand.sort(Comparator.comparing(Card::toString));
+//                        List<Card> bothPlayers = new ArrayList<>(player1Hand);
+//                        bothPlayers.addAll(player2Hand);
+//                        List<Card> bothPlayers2 = new ArrayList<>(player2Hand);
+//                        bothPlayers2.addAll(player1Hand);
+//                        String equity = calculateEquityPreFlop(player1Hand, player2Hand, new Deck()).toString();
+//                        instaResult.put(bothPlayers.toString(), equity);
+//                        instaResult.put(bothPlayers2.toString(),equity);
+//                    }
+//                }
+//            }
+//        }
+//        saveHashMapToFile(instaResult, "equity_data.ser");
+//        return instaResult;
```

Comment:
raczej jak jestes w projekcie to nie zostawaisz niczego zakomentowanego albo usuwasz albo odkomentowywujesz, ta funkcja moze sie kiedsy przydac wiec chcemy ja miec gotowa do uztywkonia w przyszlosci

Reply:
i zmien nazwe tej funkcji na calculateAndSavePreflopEqiuty()